mlir/utils/generate-test-checks.py

```diff
@@ -180,7 +180,14 @@ def process_line(line_chunks, variable_namer):
         else:
             # Otherwise, generate a new variable.
             variable = variable_namer.generate_name(ssa_name)
-            output_line += "%[[" + variable + ":.*]]"
+            if strict:
```

Comment:
Can you use a more descriptive name here? Something like `strict_name_re`, just to be a bit more clear.